Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 CoreDataModules = {editable = true,git = "https://www.github.com/AfricasVoices/CoreDataModules",ref = "v0.14.1"}
-PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "v0.0.6"}
+PipelineInfrastructure = {editable = true,git = "https://www.github.com/AfricasVoices/Pipeline-Infrastructure",ref = "add_list_cloud_blobs"}
```

Comment:
(Assuming we get a test of the 503 handler over the weekend we should do a release early next week. Testing of that is taking longer than the other automatic retry cases because it's a less common occurrence and because the version we had deployed at first had a bug in its logging that meant we don't know which cases were observed)

Reply:
great thanks will also test this on a branch over the weekend before moving automated analysis file uploads to analaysis_files_upload.py